test/aqua/runDeployedWorkers.aqua

```diff
@@ -5,15 +5,15 @@ import "@fluencelabs/aqua-lib/builtin.aqua"
 import "workers.aqua"
 import "services.aqua"
 
-export status
+export runDeployedServices
 
 data Answer:
     answer: string
     peer: string
 
-func status() -> *Answer:
+func runDeployedServices() -> *Answer:
```

Comment:
You have _the same_ code hardcoded in `src/lib/const.ts`. Any way to DRY them?

Would be very very nice to avoid hard-coding Aqua code as string in JS, because it is hard to change, compile and check hardcoded strings.

Reply:
It's not exactly the same. This function is to check `worker deploy`